modules/apps/knowledge-base/knowledge-base-service/src/main/java/com/liferay/knowledge/base/service/impl/KBArticleLocalServiceImpl.java

```diff
@@ -342,6 +350,7 @@ public void addTempAttachment(
 
 	@Override
 	public void checkKBArticles(long checkInterval) throws PortalException {
+		_checkKBArticlesByExpirationDate(new Date());
```

Comment:
There's no reason to pass in the interval; it is not used anywhere.

Reply:
This parameter will be needed to notify the user on next tasks, that is why it's here to do not change the method after.

but I can do it on the future if is better.